pkgs/applications/audio/plujain-ramp/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, pkgs  }:
+
+stdenv.mkDerivation rec {
+  name = "plujain-ramp-${version}";
+  version = "v1.1.3";
+
+  buildInputs = with pkgs; [
+    lv2
+  ];
+
+  src = fetchFromGitHub {
+    owner = "Houston4444";
+    repo = "plujain-ramp";
+    rev = "1bc1fed211e140c7330d6035122234afe78e5257";
+    sha256 = "1k7qpr8c15d623c4zqxwdklp98amildh03cqsnqq5ia9ba8z3016";
+  };
+
+  installFlags = [ "INSTALL_PATH=$(out)/lib/lv2" ];
+
+  meta = with stdenv.lib; {
+    description = "Plujain-Ramp is a mono rhythmic tremolo LV2 Audio Plugin.";
```

Comment:
The description shouldn't start with the name.

Reply:
Fixed.